Makefile

```diff
@@ -175,11 +175,3 @@ help:
 	@echo '    VERSION      The version information compiled into binaries.'
 	@echo '                 The default is obtained from git.'
 	@echo '    V            Set to 1 enable verbose build. Default is 0.'
-	@echo ''
-	@echo 'Advanced Options:'
-	@echo '    CACHE_DIR    A directory where downloaded files and other'
```

Comment:
they are but they are *very* advanced that I don't think its worth listing them here.